src/ui/components/Comments/TranscriptComments/CommentEditor/useCommentsLocalStorage.ts

```diff
@@ -0,0 +1,53 @@
+import { useGetRecordingId } from "ui/hooks/recordings";
+
+// types of editable comments:
+// - existing comments (zero or more active)
```

Comment:
what's zero or more active referring to here?

Reply:
Active/visible in the UI. Will add info.